components/overflow-group/overflow-group.js

```diff
@@ -157,28 +157,27 @@ class OverflowGroup extends OverflowGroupMixin(RtlMixin(LitElement)) {
 		}
 	}
 
-	getOverflowContainer() {
+	getOverflowContainer(overflowItems, mini) {
 		const moreActionsText = this.localize('components.overflow-group.moreActions');
-		const overflowItems = this.dropdownItems ? this.dropdownItems.slice(this.chompIndex) : [];
 		const menu = html`<d2l-dropdown-menu>
 			<d2l-menu label="${moreActionsText}">
 				${overflowItems}
 			</d2l-menu>
 		</d2l-dropdown-menu>`;
 
-		if (this.mini) {
-			return html`<d2l-dropdown-more class="${OVERFLOW_MINI_DROPDOWN_CLASS}" text="${moreActionsText}">
+		if (mini) {
+			return html`<d2l-dropdown-more class="${OVERFLOW_MINI_CLASS} d2l-overflow-dropdown-mini" text="${moreActionsText}">
```

Comment:
This class and `d2l-overflow-dropdown` are used in lp so we can't just remove them right now (but after this PR goes in we could switch them to the new OVERFLOW_CLASS and OVERFLOW_MINI_CLASS)